doc/JSON_FLAGS.md

```diff
@@ -1052,36 +1206,10 @@ Monster physical sizes.
 - ```SMALL``` Dog
 - ```TINY``` Squirrel
 
-### Special attacks
-Special attacks have been moved to [MONSTER_SPECIAL_ATTACKS.md](MONSTER_SPECIAL_ATTACKS.md) as they have all been migrated away from flags.
-
-## Mutations
-
-#### Flags
-
-- ```UNARMED_BONUS``` You get a bonus to unarmed bash and cut damage equal to unarmed_skill/2 up to 4.
-
-### Categories
 
-These branches are also the valid entries for the categories of `dreams` in `dreams.json`
+### Special attacks
 
-- ```MUTCAT_ALPHA``` "You feel...better. Somehow."
-- ```MUTCAT_BEAST``` "Your heart races and you see blood for a moment."
-- ```MUTCAT_BIRD``` "Your body lightens and you long for the sky."
-- ```MUTCAT_CATTLE``` "Your mind and body slow down. You feel peaceful."
-- ```MUTCAT_CEPHALOPOD``` "Your mind is overcome by images of eldritch horrors...and then they pass."
-- ```MUTCAT_CHIMERA``` "You need to roar, bask, bite, and flap. NOW."
-- ```MUTCAT_ELFA``` "Nature is becoming one with you..."
-- ```MUTCAT_FISH``` "You are overcome by an overwhelming longing for the ocean."
-- ```MUTCAT_INSECT``` "You hear buzzing, and feel your body harden."
-- ```MUTCAT_LIZARD``` "For a heartbeat, your body cools down."
-- ```MUTCAT_MEDICAL``` "Your can feel the blood rushing through your veins and a strange, medicated feeling washes over your senses."
-- ```MUTCAT_PLANT``` "You feel much closer to nature."
-- ```MUTCAT_RAPTOR``` "Mmm...sweet bloody flavor...tastes like victory."
-- ```MUTCAT_RAT``` "You feel a momentary nausea."
-- ```MUTCAT_SLIME``` "Your body loses all rigidity for a moment."
-- ```MUTCAT_SPIDER``` "You feel insidious."
-- ```MUTCAT_TROGLOBITE``` "You yearn for a cool, dark place to hide."
+Special attacks have been moved to [MONSTER_SPECIAL_ATTACKS.md](MONSTER_SPECIAL_ATTACKS.md) as they have all been migrated away from flags.
```

Comment:
What's the point of this note though? If the information is in the other doc just remove it from here. I don't think we need to leave breadcrumbs around like this.

Reply:
That sentence is from the old doc, I can remove the section entirely.

Reply:
Yeah just remove it

Reply:
Done, it's removed.